servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/NativeTransportUtils.java

```diff
@@ -39,6 +40,8 @@
 final class NativeTransportUtils {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NativeTransportUtils.class);
+    private static final String NETTY_NO_NATIVE_NAME = "io.netty.transport.noNative";
```

Comment:
`noNative` -> can we avoid `no` in the name which leads to double negatives and can be confusing? What about `forceNio`?

Reply:
This is the system property defined by netty, we can not change it: 
https://github.com/netty/netty/blob/1e189654f913b13be3616c552a04459465b752b3/transport-classes-epoll/src/main/java/io/netty/channel/epoll/Epoll.java#L33-L35

Reply:
Do we have to use the same property as Netty? We can decide at the ST level if we should load the libraries or not, and may want different levels of control (e.g. use io_uring vs epoll, etc.).

Reply:
`-Dio.netty.transport.noNative=true` is the only way today to enforce Java NIO for users who really need it. In order for us to be backward compatible with this use-case, we should read the same property to decide when to throw an exception. If I introduce a new property, existing users will be broken after #1979 until they find out what is the new property they have to use. We can always add our own property if we find a use-case for that.

For io_uring, a new property is added in #1971.

Reply:
I will start with netty property for now, we can add another one later if necessary